ci-scripts/integration-test-live.sh

```diff
@@ -12,11 +12,13 @@ HOST="http://127.0.0.1:$PORT"
 RPC_ADDR="127.0.0.1:$RPC_PORT"
 MODE="live"
 TEST=""
+UPDATE=""
 
 usage () {
   echo "Usage: $SCRIPT"
   echo "Optional command line arguments"
   echo "-t <string>  -- Test to run, gui or cli; empty runs both tests"
+  echo "-u <boolean> -- Update stable testdata"
```

Comment:
Why do we need to update stable data when running live tests?

Reply:
Some of the live tests use stable data where possible